assets/src/edit-story/masks/display.js

```diff
@@ -46,6 +48,9 @@ export default function WithMask({
   previewMode = false,
   ...rest
 }) {
+  // This component is used twice - random id appended to make sure
+  // id is unique for the Mask.
+  const randomId = useMemo(uuidv4, []);
```

Comment:
We're using this component in the checklist now too. The ids were matching so we needed to make sure they were a little more random.